install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -49,6 +49,7 @@ data:
             cpu: 10m
             memory: 10Mi
         securityContext:
+          runAsUser: {{ .Values.global.proxy.runAsUser }}
```

Comment:
you might as well set this as root directly.

Reply:
Done.